scripts/sct_compute_mtr.py

```diff
@@ -141,6 +155,12 @@ def get_parser():
                       mandatory=False,
                       example=['0', '1', '2'],
                       default_value='1')
+    parser.add_option(name="-omtr",
```

Comment:
i would make it `-o` as per [our convention](https://github.com/neuropoly/spinalcordtoolbox/wiki/variables#flags)

Reply:
agreed, fixed in commit [f37a7f1](https://github.com/YangHee-Min/spinalcordtoolbox/commit/f37a7f16c3093a2fdfb7b313da3396aacb4683d9)

Reply:
update description: https://github.com/neuropoly/spinalcordtoolbox/pull/2264#issue-279247920